tests/gtest/avifincrtest_helpers.cc

```diff
@@ -269,7 +287,7 @@ void DecodeIncrementally(const avifRWData& encoded_avif, bool is_persistent,
   ASSERT_NE(decoder, nullptr);
   avifDecoderSetIO(decoder.get(), &io);
   decoder->allowIncremental = AVIF_TRUE;
-  const size_t step = std::max(static_cast<size_t>(1), data.full_size / 10000);
```

Comment:
Just a side note: I found that the Style Guide also recommends using `size_t{1}` instead of `static_cast<size_t>(1)`.

> Use brace initialization to convert arithmetic types (e.g., `int64_t{x}`). This is the safest approach because code will not compile if conversion can result in information loss. The syntax is also concise.

https://google.github.io/styleguide/cppguide.html#Casting